spyder/plugins/pylint/tests/test_pylint.py

```diff
@@ -185,33 +202,47 @@ def test_pylint_widget_pylintrc(
                 for bad_name in bad_names])
 
 
-def test_pylint_max_history_conf(pylint_test_script, mocker, qtbot):
+def test_pylint_max_history_conf(pylint_test_scripts, mocker, qtbot):
```

Comment:
```suggestion
def test_pylint_max_history_conf(pylint_test_scripts, mocker):
```

Qtbot is no longer used